PowerEditor/src/ScintillaComponent/ScintillaEditView.cpp

```diff
@@ -2851,12 +2845,12 @@ void ScintillaEditView::changeCase(__inout wchar_t * const strWToConvert, const
 				if (::IsCharAlphaW(strWToConvert[i]))
 				{
 					if ((i < 1) ? true : !::IsCharAlphaNumericW(strWToConvert[i - 1]))
-						strWToConvert[i] = (WCHAR)(UINT_PTR)::CharUpperW((LPWSTR)strWToConvert[i]);
+						strWToConvert[i] = (WCHAR)(UINT_PTR)::CharUpperW(reinterpret_cast<LPWSTR>(strWToConvert[i]));

```

Comment:
Note that CharUpperW does the replacement in place, so the assignment is not necessary:
https://docs.microsoft.com/en-us/windows/win32/api/winuser/nf-winuser-charupperw
Also, the behavior for a single character isn't very clear (for me) but it seems most people use it like this.